packages/language-server/src/plugins/typescript/astro2tsx.ts

```diff
@@ -21,8 +20,8 @@ function addProps(content: string, className: string): string {
 	}
 
 	return (
-		EOL +
-		(shouldAddGlobal ? astroGlobal + EOL + astroGlobalConstDef : '') +
+		'\n' +
```

Comment:
As far as I could tell, it's not a problem to not use platform-specific line returns here. If it does, we it should be fairly easily to make a stub for it